ticketgraph/htdocs/ticketgraph.js

```diff
@@ -1,40 +1,117 @@
+var plot = null;
```

Comment:
In this case this is necessary because the `plot` var is used at a later time , but I am open to suggestions on how to handle this differently.  I am pushing new changes now